packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -73,7 +79,10 @@ export function PanelChrome({
   padding = 'md',
   title = '',
   description = '',
-  displayMode = 'default',
+  collapsible = false,
```

Comment:
Changing a component in a major way like this should probably come with some additions to tests and documentation. 

Check out https://developers.grafana.com/ui/latest/index.html?path=/story/visualizations-panelchrome--basic for our existing documentation on this, that code is https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx 

There's also a readme for the design system documentation, I believe https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/PanelChrome/PanelChrome.mdx  

FInally, we should also probably add a test that the `Collapse` component renders when appropriate.